Build/libHttpClient.141.Android.Java/build.gradle

```diff
@@ -19,3 +22,53 @@ allprojects {
         mavenCentral()
     }
 }
+
+publishing { 
+    publications { 
+        mavenAar(MavenPublication) {
+            groupId 'libHttpClientAndroid'
+            artifactId 'libHttpClient'
+            version (project.hasProperty("HCVersion") ? HCVersion : '0.0.1')
+            artifact (project.hasProperty('InputAar') ? InputAar : '../../Binaries/Release/x86/libHttpClient.141.Android.Java/libHttpClient.aar')
+
+            pom.withXml {
+                def dependencies = asNode().appendNode('dependencies')
+                project(':app').configurations.compile.allDependencies.each {
+                    if (it.group != null) {
+                        def dependency = dependencies.appendNode('dependency')
+                        dependency.appendNode('groupId', it.group)
+                        dependency.appendNode('artifactId', it.name)
+                        dependency.appendNode('version', it.version)
+                    }
+                }
+            }
+        } 
+    } 
+
+    repositories { 
+        if (project.hasProperty("PublishRemote") && (System.getenv("VSTS_ENV_ACCESS_TOKEN") != null || project.hasProperty("vstsMavenAccessToken")))
+        {
+            maven { 
+                url 'https://microsoft.pkgs.visualstudio.com/_packaging/gaming/maven/v1'
+                credentials { 
+                    username "VSTS" 
+                    //The VSTS build system will use the "VSTS_ENV_ACCCESS_TOKEN" to authenticate to VSTS feeds, otherwise it will fall back on the "vstsMavenAccessToken" property, 
+                    // and as a final fallback it will pass an invalid token so other build tasks can continue to succeed.
+                    password System.getenv("VSTS_ENV_ACCESS_TOKEN") != null ? System.getenv("VSTS_ENV_ACCESS_TOKEN") : vstsMavenAccessToken
+                } 
+            }
+        }
+        else if (System.getenv("TFS_DropLocation") != null)
+        {
+            maven
+            {
+                url ('file:///' + System.getenv("TFS_DropLocation") + '/Maven')
+            }
+        }
+        else {
+            maven {
+                url 'file:///C:/scratch/Maven'
+            }
```

Comment:
Do you intend to leave this in? Where does the path c:\scratch come from?

Reply:
You are required to set some publish path.  I changed it to set it from the command line, but just chose an arbitrary default location if its not specified

---
In reply to: [223491843](https://github.com/Microsoft/libHttpClient/pull/305#discussion_r223491843) [](ancestors = 223491843)

Reply:
can you make the default path relative to the libHttpClient location?